src/main/target/OMNIBUSF4V6/target.h

```diff
@@ -0,0 +1,189 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define USE_TARGET_CONFIG
+
+// OMNIBUSF4V6 for Version 2
+// OMNIBUSF4V61/OV61 for Public Test Version
+// (Not a valid target, .mk file must be supplied by a user)
+
+#if defined(OMNIBUSF4V6)
+#define TARGET_BOARD_IDENTIFIER "OBV6"
+#define USBD_PRODUCT_STRING "OmnibusF4 V6"
+#elif defined(OMNIBUSF4V61)
+#define TARGET_BOARD_IDENTIFIER "OV61"
+#define USBD_PRODUCT_STRING "OmnibusF4 V6Proto1"
+#endif
+
+#if defined(OMNIBUSF4V6)
```

Comment:
This is more or less a copy of the OMNIBUSF4FW target - it should be done as a variant of this target.